lib/MusicBrainz/Server/WebService/Serializer/JSON/2/CDStub.pm

```diff
@@ -32,23 +32,12 @@ __PACKAGE__->meta->make_immutable;
 no Moose;
 1;
 
-# =head1 COPYRIGHT
+=head1 COPYRIGHT AND LICENSE
 
-# Copyright (C) 2013 MetaBrainz Foundation
+Copyright (C) 2013 MetaBrainz Foundation
```

Comment:
This (and several other examples in the same commit) don't *actually* change the copyright line, but they change the style to not be commented so it shows in the diff.